docs/integration-testing.md

```diff
@@ -19,53 +18,66 @@ typically run faster. Sometimes that's not (easily) possible, especially when an
 system that is not easily mocked or stubbed such as cloud-based databases.
 
 Therefore, in many cases writing unit tests is more involved that writing an integration test, for example say we wanted
-to test our implementation of a CosmosDB-backed queue. We would have to mock the behaviour of the CosmosDB API, which -
+to test our implementation of a CosmosDB-backed queue. We would have to mock the behavior of the CosmosDB API, which -
```

Comment:
nit: spurious change?

Reply:
Yes it was a typo which I fixed but reverted now.